pkg/asset/operators/operators.go

```diff
@@ -0,0 +1,240 @@
+	assetContents := make([]asset.Content, 0)
+	templateData := &templateData{
+		AggregatorCaCert:                string(dependencies[o.aggregatorCA].Contents[certIndex].Data),
+		AggregatorCaKey:                 string(dependencies[o.aggregatorCA].Contents[keyIndex].Data),
+		ApiserverCert:                   string(dependencies[o.apiServerCertKey].Contents[certIndex].Data),
+		ApiserverKey:                    string(dependencies[o.apiServerCertKey].Contents[keyIndex].Data),
+		ApiserverProxyCert:              string(dependencies[o.apiServerProxyCertKey].Contents[certIndex].Data),
+		ApiserverProxyKey:               string(dependencies[o.apiServerProxyCertKey].Contents[keyIndex].Data),
+		Base64encodeCloudProviderConfig: "", // FIXME
+		ClusterapiCaCert:                string(dependencies[o.clusterAPIServerCertKey].Contents[certIndex].Data),
+		ClusterapiCaKey:                 string(dependencies[o.clusterAPIServerCertKey].Contents[keyIndex].Data),
+		EtcdCaCert:                      string(dependencies[o.etcdCA].Contents[certIndex].Data),
+		EtcdClientCert:                  string(dependencies[o.etcdClientCertKey].Contents[certIndex].Data),
+		EtcdClientKey:                   string(dependencies[o.etcdClientCertKey].Contents[keyIndex].Data),
+		KubeCaCert:                      string(dependencies[o.kubeCA].Contents[certIndex].Data),
+		KubeCaKey:                       string(dependencies[o.kubeCA].Contents[keyIndex].Data),
+		MachineConfigOperatorImage:      "docker.io/openshift/origin-machine-config-operator:v4.0.0",
+		McsTlsCert:                      string(dependencies[o.adminCertKey].Contents[certIndex].Data),
+		McsTlsKey:                       string(dependencies[o.adminCertKey].Contents[keyIndex].Data),
+		OidcCaCert:                      string(dependencies[o.kubeCA].Contents[certIndex].Data),
+		OpenshiftApiserverCert:          string(dependencies[o.openshiftAPIServerCertKey].Contents[certIndex].Data),
+		OpenshiftApiserverKey:           string(dependencies[o.openshiftAPIServerCertKey].Contents[keyIndex].Data),
+		OpenshiftLoopbackKubeconfig:     string(dependencies[o.kubeconfig].Contents[0].Data),
+		PullSecret:                      string(ic.PullSecret),
+		RootCaCert:                      string(dependencies[o.rootCA].Contents[certIndex].Data),
+		ServiceaccountKey:               string(dependencies[o.serviceAccountKeyPair].Contents[keyIndex].Data),
+		ServiceaccountPub:               string(dependencies[o.serviceAccountKeyPair].Contents[certIndex].Data),
+		ServiceServingCaCert:            string(dependencies[o.serviceServingCA].Contents[certIndex].Data),
+		ServiceServingCaKey:             string(dependencies[o.serviceServingCA].Contents[keyIndex].Data),
+		TectonicNetworkOperatorImage:    "quay.io/coreos/tectonic-network-operator-dev:3b6952f5a1ba89bb32dd0630faddeaf2779c9a85",
+		WorkerIgnConfig:                 "", // FIXME: this means that depending on ignition assets (risk of cyclical dependencies)
+	}
+
```

Comment:
nit: We can loop it?

Reply:
Didn't grasp what could be looped. Could do if they were all of one type, but the templates have them mixed. And more could be coming.